buildSrc/build.gradle.kts

```diff
@@ -187,12 +189,17 @@ fun Project.applyGroovyProjectConventions() {
 fun Project.applyKotlinProjectConventions() {
     apply(plugin = "kotlin")
 
-    //TODO:kotlin-dsl - uncomment after nightly upgrade
-    //apply(plugin = "org.gradle.kotlin.ktlint-convention")
+    apply(plugin = "org.gradle.kotlin.ktlint-convention")
 
     tasks.withType<KotlinCompile>().configureEach {
         kotlinOptions {
             freeCompilerArgs += listOf("-Xjsr305=strict")
         }
     }
+
+    plugins.withType<KotlinDslPlugin> {
```

Comment:
❓ `.withType<KotlinDslPlugin>().configureEach {`?

Reply:
`plugins` elements are realized as soon as they are applied, so I don't think that's necessary